packages/cli/src/modules/cloud-functions/cloud-functions.service.ts

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable security/detect-non-literal-fs-filename */
+import { CloudFunctions } from '@apillon/sdk';
+import { readFileSync } from 'fs';
+import { GlobalOptions } from '../../lib/types';
+import { paginate } from '../../lib/options';
+import { withErrorHandler } from '../../lib/utils';
+import { Storage } from '@apillon/sdk';
+
+export async function listCloudFunctions(optsWithGlobals: GlobalOptions) {
+  await withErrorHandler(async () => {
+    const data = await new CloudFunctions(optsWithGlobals).listCloudFunctions({
+      ...paginate(optsWithGlobals),
+    });
+    console.log(data.items.map((d) => d.serialize()));
+  });
+}
+
+export async function getCloudFunction(optsWithGlobals: GlobalOptions) {
+  await withErrorHandler(async () => {
+    const cloudFunction = await new CloudFunctions(optsWithGlobals)
+      .cloudFunction(optsWithGlobals.uuid)
+      .get();
+
+    cloudFunction.jobs = cloudFunction.jobs.map((job) => job.serialize());
+    console.info(cloudFunction.serialize());
+  });
+}
+
+export async function createCloudFunction(optsWithGlobals: GlobalOptions) {
+  await withErrorHandler(async () => {
+    const cloudFunctions = new CloudFunctions(optsWithGlobals);
+    const data = await cloudFunctions.createCloudFunction({
```

Comment:
if cloudFunctions.createCloudFunction fails, error is logged to console, but below logs will also be performed. You should check the value of data.